packages/amplication-e2e-tester/tests/login.test.js

```diff
@@ -0,0 +1,35 @@
+const TITLE = "Amplication";
+const LOGIN_URL = "http://localhost:3001/login";
+const HOME_PAGE_URL = "http://localhost:3001/";
+const USER_EMAIL = "lisa@simpson.com";
+const USER_PASSWORD = "secret42";
+const CONTINUE_BUTTON_CONNTENT = "Continue";

```

Comment:
*CONTENT
I recommend you to install https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker it helps me to avoid this kind of mistakes